.gitignore

```diff
@@ -1,2 +0,0 @@
-
-tempCodeRunnerFile.py
```

Comment:
This is an unrelated change / mistake introduced in a previous commit in the PR/branch. It's better to merge this commit with the previous one so the change doesn't happen in the first place.